From 1842fe2a47ee440a10ccac6d6e7b1f6eec4e0a2b Mon Sep 17 00:00:00 2001
From: razorloves <razorloves@gmail.com>
Date: Tue, 15 May 2018 23:28:25 -0500
Subject: [PATCH 3/4] wiki: Use triple backticks for code blocks instead of 8
 spaces

Triple backticks are the proper method for code blocks with Jekyll.
The use of 8 spaces was adding an unwanted space at the beginning of
each code block.

Change-Id: Ib62ffe413dc693e68403b1552b97e101966a8e0a
---
 .../before_install_jfltevzw.md                |  6 +-
 _includes/templates/recovery_install_dd.md    |  8 +--
 .../recovery_install_fastboot_generic.md      | 18 +++---
 .../recovery_install_fastboot_htc.md          | 18 +++---
 .../recovery_install_fastboot_huawei.md       | 24 ++++----
 .../recovery_install_fastboot_lenovo.md       | 20 ++++---
 .../templates/recovery_install_fastboot_lg.md | 24 ++++----
 .../recovery_install_fastboot_motorola.md     | 18 +++---
 .../recovery_install_fastboot_nexus.md        | 20 ++++---
 .../recovery_install_fastboot_oppo.md         | 20 ++++---
 .../recovery_install_fastboot_sony.md         | 56 ++++++++++---------
 .../recovery_install_fastboot_zenfone.md      |  6 +-
 .../templates/recovery_install_heimdall.md    | 11 ++--
 pages/fastboot_adb_setup.md                   | 20 +++----
 pages/meta/contributing_wiki.md               | 10 ++--
 15 files changed, 148 insertions(+), 131 deletions(-)

diff --git a/_includes/templates/device_specific/before_install_jfltevzw.md b/_includes/templates/device_specific/before_install_jfltevzw.md
index c9c8c18..86c63c5 100644
--- a/_includes/templates/device_specific/before_install_jfltevzw.md
+++ b/_includes/templates/device_specific/before_install_jfltevzw.md
@@ -11,9 +11,9 @@ In order to root your device you must first install this kernel via Heimdall.
 
     Accept the disclaimer, then insert the USB cable into the device.
 4. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window in the directory the recovery image is located, and type:
-
-        heimdall flash --BOOT boot.img
-
+```
+heimdall flash --BOOT boot.img
+```
     {% include tip.html content="The file may not be named identically to what's in this command. If the file is wrapped in a zip or tar file, extract the file first, because Heimdall isn't going to do it for you." %}
 5. A blue transfer bar will appear on the device showing the kernel being transferred, then your device will reboot.
 6. Allow it to boot and proceed with the rooting steps.
diff --git a/_includes/templates/recovery_install_dd.md b/_includes/templates/recovery_install_dd.md
index e2ad659..0a9ef26 100644
--- a/_includes/templates/recovery_install_dd.md
+++ b/_includes/templates/recovery_install_dd.md
@@ -32,10 +32,10 @@
     {% include tip.html content="The file may not be named identically to what stands in this command, so adjust accordingly." %}
    * You can use any method you are comfortable with. `adb` is universal across all devices, and works both in Android and recovery mode, providing USB debugging is enabled.
 3. Now, open an `adb shell` from a command prompt (on Windows) or terminal (on Linux or macOS) window. In that shell, type the following commands:
-
-        su
-        dd if=/sdcard/twrp-{{ twrp_codename }}.img of={{ site.data.devices[page.device].recovery_partition }}
-
+```
+su
+dd if=/sdcard/twrp-{{ twrp_codename }}.img of={{ site.data.devices[page.device].recovery_partition }}
+```
 4. Manually reboot into recovery:
     * {{ site.data.devices[page.device].recovery_boot }}
 
diff --git a/_includes/templates/recovery_install_fastboot_generic.md b/_includes/templates/recovery_install_fastboot_generic.md
index ba76e9b..3318a63 100644
--- a/_includes/templates/recovery_install_fastboot_generic.md
+++ b/_includes/templates/recovery_install_fastboot_generic.md
@@ -13,23 +13,23 @@
 {% endif %}
 2. Connect your device to your PC via USB.
 3. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 4. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 5. Flash recovery onto your device:
-
-        fastboot flash recovery twrp-x.x.x-x-{{ twrp_codename }}.img
-
+```
+fastboot flash recovery twrp-x.x.x-x-{{ twrp_codename }}.img
+```
     {% include tip.html content="The file may not be named identically to what stands in this command, so adjust accordingly." %}
 
 6. Now reboot into recovery to verify the installation:
diff --git a/_includes/templates/recovery_install_fastboot_htc.md b/_includes/templates/recovery_install_fastboot_htc.md
index 3798237..8da2f99 100644
--- a/_includes/templates/recovery_install_fastboot_htc.md
+++ b/_includes/templates/recovery_install_fastboot_htc.md
@@ -9,23 +9,23 @@ Before proceeding, ensure the data you would like to retain is backed up to your
 {% endunless %}
 2. Connect the device to your PC via USB.
 3. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 4. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 5. Now type the following command to obtain your bootloader unlock token:
-
-        fastboot oem get_identifier_token
-
+```
+fastboot oem get_identifier_token
+```
 6. Visit the [HTCDev Bootloader Unlock](http://www.htcdev.com/bootloader/) website and follow the instructions there to obtain your unlock key and unlock your bootloader. If your device does not appear in the drop-down list, select **All Other Supported Models**.
 
     {% include note.html content="If your device is not supported by the HTCDev Bootloader Unlock website, you will need to use an alternative bootloader unlock method like [SunShine](http://theroot.ninja/)." %}
diff --git a/_includes/templates/recovery_install_fastboot_huawei.md b/_includes/templates/recovery_install_fastboot_huawei.md
index aef7ab8..0725fac 100644
--- a/_includes/templates/recovery_install_fastboot_huawei.md
+++ b/_includes/templates/recovery_install_fastboot_huawei.md
@@ -8,31 +8,31 @@ Before proceeding, ensure the data you would like to retain is backed up to your
 2. Follow the instructions and get your unlock password.
 3. Connect your device to your PC via USB.
 4. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 5. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 6. Now type the following command to unlock the bootloader:
-
-        fastboot oem unlock ****************
-
+```
+fastboot oem unlock ****************
+```
     Replace **************** with the 16-digit unique unlock password that was obtained in step 2.
 7. Wait for the bootloader unlocking process to complete. Once finished, your device will reboot to system and will be restored to factory settings.
     If an incorrect password is entered, a message will be displayed and your device will also reboot to system.
 8. Since the device resets completely, you will need to re-enable USB debugging to continue.
 9. To check the bootloader lock status, reboot the device to fastboot mode (repeating steps 3 and 4) and type:
-
-        fastboot oem get-bootinfo
-
+```
+fastboot oem get-bootinfo
+```
     The message `Bootloader Lock State: LOCKED` indicates that the bootloader is still locked. Perform the unlocking procedure again and check that the password was entered correctly.
     On the other hand, the message `Bootloader Lock State: UNLOCKED` indicates that the bootloader has been unlocked. You can now install third-party firmware.
 
diff --git a/_includes/templates/recovery_install_fastboot_lenovo.md b/_includes/templates/recovery_install_fastboot_lenovo.md
index d40e0be..2377bbf 100644
--- a/_includes/templates/recovery_install_fastboot_lenovo.md
+++ b/_includes/templates/recovery_install_fastboot_lenovo.md
@@ -13,25 +13,29 @@ Before proceeding, ensure the data you would like to retain is backed up to your
 {% endunless %}
 6. Connect the device to your PC via USB.
 7. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 8. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 9. Now type the following command to unlock the bootloader:
 
 {% if site.data.devices[page.device].custom_unlock_cmd %}
-        {{ site.data.devices[page.device].custom_unlock_cmd }}
+    ```
+{{ site.data.devices[page.device].custom_unlock_cmd }}
+    ```
 {% else %}
-        fastboot oem unlock
+    ```
+fastboot oem unlock
+    ```
 {% endif %}
 
 10. If the device doesn't automatically reboot, reboot it. It should now be unlocked.
diff --git a/_includes/templates/recovery_install_fastboot_lg.md b/_includes/templates/recovery_install_fastboot_lg.md
index 3de4bfb..74138f9 100644
--- a/_includes/templates/recovery_install_fastboot_lg.md
+++ b/_includes/templates/recovery_install_fastboot_lg.md
@@ -8,28 +8,28 @@ Before proceeding, ensure the data you would like to retain is backed up to your
 2. Follow the instructions and get your unlock file.
 3. Connect your device to your PC via USB.
 4. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 5. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 5. Now type the following command to unlock the bootloader:
-
-        fastboot flash unlock unlock.bin
-
+```
+fastboot flash unlock unlock.bin
+```
     Where `unlock.bin` is the bootloader unlock file you received in the email.
 7. Wait for the bootloader unlocking process to complete. Once finished, you can check if bootloader is successfully unlocked by typing:
-
-        fastboot getvar unlocked
-
+```
+fastboot getvar unlocked
+```
     Verify that the response is `unlocked: yes`. In that case, you can now install third-party firmware.
 8. Since the device resets completely, you will need to re-enable USB debugging to continue.
 
diff --git a/_includes/templates/recovery_install_fastboot_motorola.md b/_includes/templates/recovery_install_fastboot_motorola.md
index b1c7826..ef4e61b 100644
--- a/_includes/templates/recovery_install_fastboot_motorola.md
+++ b/_includes/templates/recovery_install_fastboot_motorola.md
@@ -6,23 +6,23 @@ Before proceeding, ensure the data you would like to retain is backed up to your
 
 1. Connect the device to your PC via USB.
 2. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 3. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 4. Now type the following command to get the bootloader status:
-
-        fastboot oem device-info
-
+```
+fastboot oem device-info
+```
 5. Follow the instructions at [Motorola Support](http://motorola-global-portal.custhelp.com/app/standalone/bootloader/unlock-your-device-a) to unlock your bootloader.
 
     {% include note.html content="If your device is not supported by the Motorola Bootloader Unlock website, you will need to use an alternative bootloader unlock method like [SunShine](http://theroot.ninja/)." %}
diff --git a/_includes/templates/recovery_install_fastboot_nexus.md b/_includes/templates/recovery_install_fastboot_nexus.md
index 64ec9e0..0962e17 100644
--- a/_includes/templates/recovery_install_fastboot_nexus.md
+++ b/_includes/templates/recovery_install_fastboot_nexus.md
@@ -9,25 +9,29 @@ Before proceeding, ensure the data you would like to retain is backed up to your
 {% endunless %}
 2. Connect the device to your PC via USB.
 3. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 4. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 5. Now type the following command to unlock the bootloader:
 
 {% if site.data.devices[page.device].custom_unlock_cmd %}
-        {{ site.data.devices[page.device].custom_unlock_cmd }}
+    ```
+{{ site.data.devices[page.device].custom_unlock_cmd }}
+    ```
 {% else %}
-        fastboot oem unlock
+    ```
+fastboot oem unlock
+    ```
 {% endif %}
 
 6. If the device doesn't automatically reboot, reboot it. It should now be unlocked.
diff --git a/_includes/templates/recovery_install_fastboot_oppo.md b/_includes/templates/recovery_install_fastboot_oppo.md
index 98124e7..783634a 100644
--- a/_includes/templates/recovery_install_fastboot_oppo.md
+++ b/_includes/templates/recovery_install_fastboot_oppo.md
@@ -36,25 +36,29 @@ Before proceeding, ensure the data you would like to retain is backed up to your
 {% endunless %}
 11. Connect the device to your PC via USB.
 12. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 13. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 14. Now type the following command to unlock the bootloader:
 
 {% if site.data.devices[page.device].custom_unlock_cmd %}
-        {{ site.data.devices[page.device].custom_unlock_cmd }}
+    ```
+{{ site.data.devices[page.device].custom_unlock_cmd }}
+    ```
 {% else %}
-        fastboot oem unlock
+    ```
+fastboot oem unlock
+    ```
 {% endif %}
 
 15. If the device doesn't automatically reboot, reboot it. It should now be unlocked.
diff --git a/_includes/templates/recovery_install_fastboot_sony.md b/_includes/templates/recovery_install_fastboot_sony.md
index 0dd94c2..a660cf3 100644
--- a/_includes/templates/recovery_install_fastboot_sony.md
+++ b/_includes/templates/recovery_install_fastboot_sony.md
@@ -18,18 +18,18 @@ If you wish to backup the TA partition first, you can find tutorials related to
 2. Go to `service info` > `configuration` and check `rooting status` - you can only continue if it says `Bootloader unlock allowed: Yes`.
 3. Connect the device to your PC via USB.
 4. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 5. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 6. Follow the instructions on [Sony's official unlocking website](http://developer.sonymobile.com/unlockbootloader/unlock-yourboot-loader/) to unlock your bootloader.
 7. Since the device resets completely, you will need to re-enable USB debugging to continue.
@@ -51,40 +51,44 @@ If you wish to backup the TA partition first, you can find tutorials related to
 {% endif %}
 2. Connect your device to your PC via USB.
 3. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        adb reboot bootloader
-
+```
+adb reboot bootloader
+```
     {% if site.data.devices[page.device].download_boot %}
     You can also boot into fastboot mode via a key combination:
 
     * {{ site.data.devices[page.device].download_boot }}
     {% endif %}
 4. Once the device is in fastboot mode, verify your PC finds it by typing:
-
-        fastboot devices
-
+```
+fastboot devices
+```
     {% include tip.html content="If you see `no permissions fastboot` while on Linux or macOS, try running `fastboot` as root." %}
 5. Temporarily flash TWRP to `boot`:
-
-        fastboot flash boot twrp-x.x.x-x-{{ twrp_codename }}.img
-
+```
+fastboot flash boot twrp-x.x.x-x-{{ twrp_codename }}.img
+```
     {% include tip.html content="The file may not be named identically to what stands in this command, so adjust accordingly. Remember to adjust the filename in the following commands as well." %}
 6. Reboot to the TWRP recovery:
-
-        fastboot reboot
+```
+fastboot reboot
+```
 7. Push the TWRP image to your device:
-
-        adb push twrp-x.x.x-x-{{ twrp_codename }}.img /sdcard
+```
+adb push twrp-x.x.x-x-{{ twrp_codename }}.img /sdcard
+```
 8. Enter shell on the device:
-
-        adb shell
+```
+adb shell
+```
 9. Flash TWRP to `recovery` permanently:
-
-        dd if=/sdcard/twrp-x.x.x-x-{{ twrp_codename }}.img of=/dev/block/platform/msm_sdcc.1/by-name/FOTAKernel
+```
+dd if=/sdcard/twrp-x.x.x-x-{{ twrp_codename }}.img of=/dev/block/platform/msm_sdcc.1/by-name/FOTAKernel
+```
 10. Exit the adb shell:
-
-        exit
-
+```
+exit
+```
 {% else %}
 
 {% include templates/recovery_install_fastboot_generic.md %}
diff --git a/_includes/templates/recovery_install_fastboot_zenfone.md b/_includes/templates/recovery_install_fastboot_zenfone.md
index bd8d7d4..0341a24 100644
--- a/_includes/templates/recovery_install_fastboot_zenfone.md
+++ b/_includes/templates/recovery_install_fastboot_zenfone.md
@@ -15,9 +15,9 @@
 2. Connect the device to your PC via USB.
 3. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window and change to the location where the Unlock Device App `.apk` resides.
 4. Install the `.apk` by typing:
-
-        adb install Name_of_unlock_app.apk
-
+```
+adb install Name_of_unlock_app.apk
+```
     * For example, on the ZE551ML device, it would be `adb install UnlockApp_ze551ml_20150723.apk`.
 
 5. Run the application **Unlock Device Tool** from the device and agree to the terms.
diff --git a/_includes/templates/recovery_install_heimdall.md b/_includes/templates/recovery_install_heimdall.md
index c2ef8eb..804c6f3 100644
--- a/_includes/templates/recovery_install_heimdall.md
+++ b/_includes/templates/recovery_install_heimdall.md
@@ -30,8 +30,9 @@ The preferred method of installing a custom recovery is through this boot mode{%
     4. Click **Replace Driver** (having to select **Install Driver** from the drop down list built into the button).
     5. If you are prompted with a warning that the installer is unable to verify the publisher of the driver, select **Install this driver anyway**. You may receive two more prompts about security. Select the options that allow you to carry on.
 5. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window, and type:
-
-        heimdall print-pit
+```
+heimdall print-pit
+```
 6. If the device reboots, Heimdall is installed and working properly.
 
 {% if custom_downgrade_instructions %}
@@ -56,9 +57,9 @@ The preferred method of installing a custom recovery is through this boot mode{%
 
     Accept the disclaimer, then insert the USB cable into the device.
 4. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window in the directory the recovery image is located, and type:
-
-        heimdall flash --RECOVERY twrp-x.x.x-x-{{ twrp_codename }}.img --no-reboot
-
+```
+heimdall flash --RECOVERY twrp-x.x.x-x-{{ twrp_codename }}.img --no-reboot
+```
     {% include tip.html content="The file may not be named identically to what stands in this command, so adjust accordingly. If the file is wrapped in a zip or tar file, extract the file first, because Heimdall is not going to do it for you." %}
 5. A blue transfer bar will appear on the device showing the recovery being transferred.
 6. Unplug the USB cable from your device.
diff --git a/pages/fastboot_adb_setup.md b/pages/fastboot_adb_setup.md
index 748c67a..2570099 100644
--- a/pages/fastboot_adb_setup.md
+++ b/pages/fastboot_adb_setup.md
@@ -39,22 +39,22 @@ Google hosts zips including only `adb` and `fastboot`. You can set these up for
 1. Download the [macOS zip](https://dl.google.com/android/repository/platform-tools-latest-darwin.zip) from Google.
 2. Extract it somewhere - for example, `~/adb-fastboot`.
 3. Add the following to `~/.bash_profile`:
-
-        if [ -d "$HOME/adb-fastboot/platform-tools" ] ; then
-            export PATH="$HOME/adb-fastboot/platform-tools:$PATH"
-        fi
-
+```sh
+if [ -d "$HOME/adb-fastboot/platform-tools" ] ; then
+    export PATH="$HOME/adb-fastboot/platform-tools:$PATH"
+fi
+```
 4. Log out and back in.
 
 ### On Linux
 1. Download the [Linux zip](https://dl.google.com/android/repository/platform-tools-latest-linux.zip) from Google.
 2. Extract it somewhere - for example, `~/adb-fastboot`.
 3. Add the following to `~/.profile`:
-
-        if [ -d "$HOME/adb-fastboot/platform-tools" ] ; then
-            export PATH="$HOME/adb-fastboot/platform-tools:$PATH"
-        fi
-
+```sh
+if [ -d "$HOME/adb-fastboot/platform-tools" ] ; then
+    export PATH="$HOME/adb-fastboot/platform-tools:$PATH"
+fi
+```
 4. Log out and back in.
 5. You may also need to set up udev rules: see [this repository](https://github.com/M0Rf30/android-udev-rules#installation) for more info.
 
diff --git a/pages/meta/contributing_wiki.md b/pages/meta/contributing_wiki.md
index 4c72ece..b2d0f87 100644
--- a/pages/meta/contributing_wiki.md
+++ b/pages/meta/contributing_wiki.md
@@ -77,12 +77,12 @@ git commit
 
 An editor will pop up. In the first line, type a short (below 80 character) description of your changes,
 then put a blank line, and, if you want, a more detailed description of your changes. For example:
+```
+Add the contributing page
 
-        Add the contributing page
-
-        The contributing page will show people how they can edit our wiki, which
-        is important, since it wouldn't be much of a wiki without editors.
-
+The contributing page will show people how they can edit our wiki, which
+is important, since it wouldn't be much of a wiki without editors.
+```
 Save the file and exit the editor. Now upload your changes to GitHub:
 
 ```
-- 
2.17.1

